ah/kubernetes_workers.go

```diff
@@ -0,0 +1,51 @@
+package ah
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+// KubernetesWorker object
+type KubernetesWorker struct {
+	Labels           map[string]string `json:"labels,omitempty"`
+	ID               string            `json:"id,omitempty"`
+	Name             string            `json:"name,omitempty"`
+	State            string            `json:"state,omitempty"`
+	Type             string            `json:"type,omitempty"`
+	CreatedAt        string            `json:"created_at,omitempty"`
+	ExternalIpID     string            `json:"external_ip_id,omitempty"`
+	PrivateNetworkID string            `json:"private_network_id,omitempty"`
+	CloudServerID    string            `json:"cloud_server_id,omitempty"`
+}
+
+type ClusterDeleteNodeRequest struct {
```

Comment:
I think we should use either workers or nodes everywhere, I assume once it's public api, it should be `ClusterDeleteWorkerRequest`